functions/tests/test_stripe_create_account.py

```
@@ -0,0 +1,451 @@
+"""
+Comprehensive end-to-end tests for Stripe account creation functionality.
+Tests the complete account creation workflow including validation,
+Stripe account management, and error handling scenarios.
+"""
+
+import json
+from unittest.mock import Mock, patch
+
+import pytest
+import stripe
```

Ruff (0.12.2)

`stripe` imported but unused

Remove unused import: `stripe`

(F401)